backend/tracim_backend/lib/search/elasticsearch_search/es_models.py

```diff
@@ -34,27 +34,54 @@
 edge_ngram_token_filter = analysis.token_filter(
     "edge_ngram_filter", type="edge_ngram", min_ngram=2, max_gram=20
 )
+# NOTE 2021-11-02 - S.G. - Configuring a maximum length for tokens (e.g. words)
+# so that the following error does not occur:
+# https://stackoverflow.com/questions/24019868/utf8-encoding-is-longer-than-the-max-length-32766
+# It can happen in HTML custom properties as images/video are embedded in it
```

Comment:
```suggestion
# It can happen in HTML custom properties in which images or videos are embedded
```
(I was incorrectly parsing images/video as a mimetype when reading this comment)